recipes/piqs/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "piqs" %}
+{% set version = "1.2" %}
+{% set sha256 = "2be0f330ebf5cdad5c353c763d683054f09c6a9d08f8b34a71a056c29755cd75" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/nathanshammah/piqs/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python >=3.5
+    - pip
+    - setuptools
+    - numpy 1.8
+    - cython >=0.21
+
+  run:
+    - python >=3.5
+    - numpy >=1.8
+    - cython >=0.21
+    - scipy >=0.15
+    - qutip >=4.2
+
+test:
+ imports:
+   - piqs
+   - piqs.tests
+
+about:
+  home: https://github.com/nathanshammah/piqs
+  license: BSD
+  license_family: BSD
+  license_file: LICENSE
+  summary: 'Efficient numerical simulation of Lindblad dynamics using permutational invariance'
+  doc_url: http://piqs.readthedocs.io/
+  dev_url: https://github.com/nathanshammah/piqs
+
+extra:
+  recipe-maintainers:
+    - sahmed95
+    - nathanshammah
```

Comment:
Are you ok being a maintainer, @nathanshammah?

Reply:
yes, thank you.